ada/extensions/src/libadalang-helpers.adb

```diff
@@ -36,6 +36,29 @@ package body Libadalang.Helpers is
                  renames To_Unbounded_String;
    function "+" (S : Unbounded_String) return String renames To_String;
 
+   procedure Print_Error (Message : String);
+   --  Shortcut for Put_Line (Standard_Error, Message)
```

Comment:
According to the below implem, the comment here is misleading: the implem is more than just a shortcut for `Put_Line (Standard_Error, Message)`. What about just saying `Helper to print an error on stderr` if you don't want to get into the details.

Reply:
Right. It used to be a real shortcut, but I later noticed that GNATCOLL.Projects.Load already includes line breaks in its error message through its Error callback… and Ada.Text_IO does not play nicely with that… Anyway, I’ll update the comment, thanks!